.ruby-version

```diff
@@ -0,0 +1 @@
+2.0.0
```

Comment:
Lets bump this up to `ruby-2.4.1`. Ruby 2 is no longer maintained and `ruby_dep` needs at least 2.2.5 to install.